app-text/wordgrinder/wordgrinder-0.7.1.recipe

```diff
@@ -33,7 +34,11 @@ REQUIRES="
 	lib:libncurses$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
 	"
-
+if [ -n "$secondaryArchSuffix" ]; then
+CONFLICTS="
+	wordgrinder_dictionaries
+	"
+fi
 BUILD_REQUIRES="
```

Comment:
We like to have a blank line before  BUILD_REQUIRES.
Source: [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)